storage/remote/queue_manager.go

```diff
@@ -375,12 +404,43 @@ func (t *QueueManager) Start() {
 	go t.reshardLoop()
 }
 
+// loadCheck will check if the SegmentRecord.json
+// contents and segment is valid
+func (t *QueueManager) loadCheck() {
+	// Checking if the JSON file is corrupted
```

Comment:
All comments should be full sentences, so they should start with a capital and end with a period. One exception is comments for functions whose first letter is not a capital (such as the function comment above).